src/basic/mkdir.h

```diff
@@ -26,9 +23,7 @@ int mkdir_parents_safe(const char *prefix, const char *path, mode_t mode, uid_t
 int mkdir_p(const char *path, mode_t mode);
 int mkdir_p_safe(const char *prefix, const char *path, mode_t mode, uid_t uid, gid_t gid, MkdirFlags flags);
 int mkdir_p_root_full(const char *root, const char *p, uid_t uid, gid_t gid, mode_t m, usec_t ts, char **subvolumes);
-static inline int mkdir_p_root(const char *root, const char *p, uid_t uid, gid_t gid, mode_t m) {
-        return mkdir_p_root_full(root, p, uid, gid, m, USEC_INFINITY, NULL);
```

Comment:
USEC_INFINITY is defined in forward.h, hence I do not think it is necessary to make this non-inline.